nitro-protocol/contracts/test/TESTNitroUtils.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.7.6;
+pragma experimental ABIEncoderV2;
+
+import '../interfaces/INitroTypes.sol';
+import {NitroUtils} from '../libraries/NitroUtils.sol';
+
+
+/**
+ * @dev This contract extends the NitroUtils contract to enable it to be more easily unit-tested. It exposes public or external functions call into internal functions. It should not be deployed in a production environment.
+ */
+contract TESTNitroUtils {
+
+
+    /**
+     * @dev Wrapper for otherwise internal function. Given a digest and digital signature, recover the signer
```

Comment:
```suggestion
contract TESTNitroUtils {

    /**
     * @dev Wrapper for otherwise internal function. Given a digest and digital signature, recover the signer
```